config/environment.js

```diff
@@ -7,10 +7,18 @@ function isTrue(value) {
 
 module.exports = function(environment) {
   var devicefarmEnv = process.env.IRENE_DEVICEFARM_URL || "wss://devicefarm.appknox.com";
+  var deviceFarmPath = "/websockify";
   var deviceFarmWebsockifyHost = url.parse(devicefarmEnv);
   var deviceFarmSsl = deviceFarmWebsockifyHost.protocol == "wss:";
   var deviceFarmPort = deviceFarmWebsockifyHost.port || (deviceFarmSsl ? 443:80);
   var deviceFarmHost = deviceFarmWebsockifyHost.hostname;
+  var deviceFarmURL = url.format(
+    {
+      protocol: deviceFarmWebsockifyHost.protocol,
+      hostname: deviceFarmHost,
+      port: (deviceFarmSsl ? 443:80),
```

Comment:
If the port is specified along with host, this will not accept that. Use `deviceFarmPort` here